velox/exec/Spiller.cpp

```diff
@@ -60,6 +58,10 @@ SpillerBase::SpillerBase(
           spillStats,
           spillConfig->fileCreateConfig) {
   TestValue::adjust("facebook::velox::exec::SpillerBase", this);
+  compareFlags_.reserve(sortingKeys.size());
```

Comment:
Can we make this a const member and provide an anonymous function to convert from sortkeys to compare flags? Also we should have sanity check that the sort key channels are prefix of input?